incubator/cassandra/templates/cassandra-statefulset.yaml

```diff
@@ -60,6 +60,29 @@ spec:
 {{- if .Values.selector }}
 {{ toYaml .Values.selector | indent 6 }}
 {{- end }}
+      serviceAccountName: {{ template "cassandra.fullname" . }}
```

Comment:
This adds a service account that may not exist.

Reply:
right actually i did remove that in my local setup already, i will push an update in a few minutes